assets/components/CheckboxInput.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { gettext } from 'utils';
+
+
+const CheckboxInput = ({name, label, onChange, value}) => {
+    return (
+        <div className='form-check'>
+            <label htmlFor="{name}" className='form-check-label'>
+                <input
+                    type="checkbox"
+                    name={name}
+                    className="form-check-input"
+                    checked={value}
+                    onChange={onChange} />
+                {gettext(label)}</label>
```

Comment:
this would not pickup the string for translation, it's better to use gettext the other way around:
```jsx
<CheckboxInput label={gettext('Label')} />
```